sysbench/scripts/run_all.sh

```diff
@@ -1,5 +1,5 @@
 
-if [ $# != 9 ]; then
+if [ $# != 10 ]; then
```

Comment:
this means that we must input 10 args to run this? 
Can we support paring argument and using default value?